assemblies/assembly-planner-java.adoc

```diff
@@ -0,0 +1,750 @@
+    <scope>test</scope>
+  </dependency>
+
+  <dependency>
+    <groupId>ch.qos.logback</groupId>
+    <artifactId>logback-classic</artifactId>
+    <version>1.2.3</version>
+  </dependency>
+----
++
+The following example shows the complete `pom.xml` file.
++
+[source,xml,subs=attributes+]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.acme</groupId>
+  <artifactId>optaplanner-hello-world-school-timetabling-quickstart</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <properties>
+    <maven.compiler.release>{java-version}</maven.compiler.release>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.optaplanner</groupId>
+        <artifactId>optaplanner-bom</artifactId>
+        <version>{project-version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <dependency>
+        <groupId>ch.qos.logback</groupId>
+        <artifactId>logback-classic</artifactId>
+        <version>1.2.3</version>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.optaplanner</groupId>
+      <artifactId>optaplanner-core</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-classic</artifactId>
+      <scope>runtime</scope>
+    </dependency>
+
+    <!-- Testing -->
+    <dependency>
+      <groupId>org.optaplanner</groupId>
+      <artifactId>optaplanner-test</artifactId>
+      <scope>test</scope>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>exec-maven-plugin</artifactId>
+        <version>3.0.0</version>
+        <configuration>
+          <mainClass>org.acme.schooltimetabling.TimeTableApp</mainClass>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+</project>
+----
+** For Gradle, add the following dependencies to the `gradle.build` file:
++
+[source]
+----
+dependencies {
+    implementation "org.optaplanner:optaplanner-core:${optaplannerVersion}"
+    runtimeOnly "ch.qos.logback:logback-classic:${logbackVersion}"
+
+    testImplementation "org.optaplanner:optaplanner-test:${optaplannerVersion}"
+}
+----
++
+The following example shows the completed `gradle.build` file.
++
+[source,groovy,subs=attributes+]
+----
+plugins {
+    id "java"
+    id "application"
+}
+
+def optaplannerVersion = "{project-version}"
+def logbackVersion = "1.2.3"
+
+group = "org.acme"
+version = "0.1.0-SNAPSHOT"
+
+repositories {
+    mavenCentral()
```

Comment:
Just for your information
The maven version would be different if we do not add 
```
repositories {
    mavenCentral()
    mavenLocal()
    maven {
        url "https://repository.jboss.org/nexus/content/groups/public/"
        mavenContent {
            snapshotsOnly()
        }
    }
}
```
The point is that in maven version we use dependency management which turns to have a optaplanner-parent  that use repository.jboss.org
So when user uses the snapshot version which was not yet released like 8.12.0-SNAPSHOT the maven version will work and gradle will fail to find the artifact in maven repo. Because 8.12.0-SNAPSHOT was build but not released